src/content/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes/browser-agent-v1.237.1.mdx

```diff
@@ -0,0 +1,32 @@
+---
+subject: Browser agent
+releaseDate: "2023-08-02"
+version: 1.237.1
+features: []
+bugs: ["SPA feature respects ajax deny_list for fetch","prevent dollar symbol only variable names","Undefined deny list ignores block internal","Fix serialization issue with array data"]
+security: []
+---
+
+## v1.237.1
+
+### Bug Fixes
+
+#### SPA feature respects ajax deny_list for fetch
+The SPA feature will now properly respect the ajax deny list and block internal settings for fetch calls. In prior versions, the SPA feature only consulted the ajax deny list for XMLHttpRequest calls.
+
+#### prevent dollar symbol only variable names
+Prevent the build process of the agent from creating minified variable names that only contain the dollar symbol. If such a variable is immediately followed by an ampersand, as could be the case in a minified logic block, it could be picked up by  string replacement logic applied to outgoing HTML within their server.
```

Comment:
```suggestion
Prevent the build process of the agent from creating minified variable names that only contain the dollar symbol. If such a variable is immediately followed by an ampersand, as could be the case in a minified logic block, it could be picked up by string replacement logic applied to outgoing HTML within their server.
```